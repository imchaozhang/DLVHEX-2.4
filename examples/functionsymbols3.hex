p(f(a)).
limit(f(a)).
limit(f(aa)).
p(f(Y)) :- p(f(X)), &testConcat[X,a](Y), limit(f(X)). 
