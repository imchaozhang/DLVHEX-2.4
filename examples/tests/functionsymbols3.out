{p(f(a)),limit(f(a)),limit(f(aa)),p(f(aa)),p(f(aaa))}
